tools/autograd/derivatives.yaml

```diff
@@ -94,6 +94,22 @@
 # used by our frontend.  You MUST check the frontend code; search for
 # OpName.apply to see if it's still using a legacy Python style API.
 #
+# Note: Returning views.
+# The following cases exist:
+#     - If a function returns no view, it can have arbitrary outputs.
+#     - If a function return at least one Tensor that is a differentiable view
+#       of one of its input:
+#         - If there is only one differentiable output, this Tensor is marked as a
+#           differentiable view. (alias or transpose for example)
+#         - If there are more than one differentiable output, by default all the views are
+#           marked as differentiable views and created with allow_rebase_history=false.
+#           Meaning that any inplace operation on it will raise an error. (unbind for example)
+#         - If there are more than one differentiable output, and the function is only doing
+#           view operation, it can be added to PURE_VIEW_FUNCTIONS in gen_autograd.py. In that
+#           case, the outputs won't be created with allow_rebase_history=false but the backward
+#           specified in this file might NOT be used. See the note in gen_autograd.py for
+#           more details. (split for example)
```

Comment:
Similarly, this comment doesn't make it clear why unbind doesn't satisfy the critera for "pure view functions".  It's only doing a view operation!